docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
-- **Hash**: distributes data across different nodes based on a hash algorithm.
+- **Round-Robin**: distributes data across different nodes in a cyclic.
+- **Range**: distributes data across different nodes based on the ranges of partitioning column values. As shown in the diagram, the ranges [1-3] and [4-6] correspond to different nodes.
+- **List**: distributes data across different nodes based on the discrete values of partitioning columns, such as gender and province. Each discrete value is mapped to a node, and multiple different values might be mapped to the same node.
+- **Hash**: distributes data across different nodes based on a hash function.
 
-To achieve more flexible data distribution, you can combine the preceding distribution methods based on your business requirements, such as hash-hash, range-hash, and hash-list.
+To achieve more flexible data partitioning, in addition to using one of the above data distribution methods, you can also combine these methods based on specific business requirements. Common combinations include Hash+Hash, Range+Hash, and Hash+List.
 
 ### Distribution methods in StarRocks
 
-StarRocks uses a flexible two-level distribution method that combines partitioning and bucketing.
+StarRocks supports both seperate and composite use of data distribution methods.
 
-- The first level is partitioning: supports Range distribution and no partitioning (the entire table is regarded as one partition).
+> **NOTE**
+>
+> In addition to the general distribution methods, StarRocks also supports Random distribution to simplify bucketing configuration.
+
+Also, StarRocks distributes data by implementing the two level partitioning + bucketing method.
+
+- The first level is partitioning: Data within a table can be partitioned. The partitioning method can be range partitioning or list partitioning. Or you can choose not to use partitioning (the entire table is regarded as one partition).
+- The second level is bucketing: Data in a partition needs to be further distributed into smaller bucktes. The bucketing method can be hash or random bucketing.
+
+| **Distribution method**   | **Partitioning and bucketing method**                        | **Description**                                              |
+| ------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Random distribution       | Random bucketing                                             | The entire table is considered a partition. Data in the table is randomly distributed into different buckets. This is the default data distribution method. |
+| Hash distribution         | Hash bucketing                                               | The entire table is considered a partition. The data in the table is distributed to the corresponding bucket, which is based on the hash value of the data's bucketing key by using a hashing function. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/568a9ec6-36a7-4ae4-9547-a6c62776e110)
-> a hash function. See to it throughout the docs.